package.json

```diff
@@ -11,7 +11,7 @@
     "license": "MIT",
     "scripts": {
         "serve": "vue-cli-service serve",
-        "build": "vue-cli-service build",
+        "build": "vue-cli-service build && node deploy.js",
```

Comment:
```suggestion
        "build": "vue-cli-service build && rm -r ./docs && cp -r ./dist ./docs",
```

Reply:
And also
```
echo "<domain>" > CNAME
```

Reply:
This looks much cleaner! It doesn't work on Windows, but I guess we don't have to build on Windows anyways. I will add the change once I test it on Ubuntu (WSL)